test/elm/query/query-test.ts

```diff
@@ -433,27 +433,27 @@ describe('AggregateQuery', () => {
       new Interval(new DateTime(1982, 3, 16, 15, 0), new DateTime(2013, 5, 23, 10, 0))
     ];
     (await this.expressionStartingAggregation.exec(this.ctx)).should.eql(ret);
-    should(this.ctx.localId_context[30]).not.be.undefined();
+    should(this.ctx.localId_context[245]).not.be.undefined();
```

Comment:
More localIds that are specified as numbers even though `localId_context` keys are technically strings (and so are localIds). And another place we could maybe just make a helper function to grab the localId from the data so we don't need to keep updating these tests.